### DISCUSSION TITLE
unknown command from argument 3: `custom-tlv`

### DISCUSSION THREAD

**Discussion by superPN** [lldpd/lldpd#632] [Q&A]
Hi,
I dont have a write right when I run lldpcli (as root).
`
root@ubuntu:~# lldpcli
[lldpcli] # configure lldp custom-tlv oui 00,21,6C subtype 1 oui-info CE,06,0C,00,E9,FD,02,03,15,CD,5B,07
2024-01-25T10:56:24 [WARN/lldpctl] unknown command from argument 3: `custom-tlv`
`
My config :
Ubuntu 14.04
I downloaded lldpd using "sudo apt-get install lldpd"
lldpd -version 0.7.7

I can make sure I have root privileges when executing this command.
Before this, I also encountered such a problem when using the "configure" command. At that time, I executed `chmod u-s /usr/sbin/lldpctl` and solved the problem.(https://github.com/lldpd/lldpd/issues/56)
 But now I can't find the problem about "custom-tlv" Any solution.

Thanx for your work

**Comment by vincentbernat** [comment]
This version is super old and does not support custom TLV.
